packages/connect-popup/e2e/tests/popup-close.test.ts

```diff
@@ -196,6 +200,7 @@ test.beforeAll(async () => {
     // reloading page might end with "closed device" error from here:
     // https://github.com/trezor/trezord-go/blob/106e5e9af3573ac2b27ebf2082bbee91650949bf/usb/libusb.go#L511
     // this was probably targeted by this (never merged) trezor-bridge PR https://github.com/trezor/trezord-go/pull/156
+    // TODO: investigate if this is still an issue
```

Comment:
I think that this might have been fixed by transport-refactor-2 branch. The problem was sending post(cancel) and receive when it was not allowed. I totally wouldn't mind if you investigate :D maybe you could try to enable these skipped tests in another branch. Totally optional of course.

Reply:
What this test is testing is what happens when the host (connect-explorer in this case) is reloaded when popup is still working.
I think in that case popup should display the error page. But it is not implemented like that right now. But That is what I think it should happen. WDYT?

Reply:
The problem is, that when host is reloaded, popup does not know that it happened. We would need to do either:
1. host would need to send message in "onBeforeUnload" 
2. or there would need to be some ping-pong keep alive message exchanges between host and popup

all in all this feels to be out of scope of this PR

Reply:
Yes, totally out of scope of this PR but interesting to think about.